### PR TITLE
KAA-1148: QA Review fixes

### PR DESCRIPTION
**Pull request by RostakaGmfun**
/cc @Polinaa

### CODE REVIEW

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Linux/index.md [hunk 1]
```diff
@@ -17,7 +17,6 @@ This page describes Kaa C++ SDK build process on Linux system.
 **Host OS:** Ubuntu 14.04 LTS Desktop 64-bit.
 
 ## Installing prerequisites
-
 **NOTE:** Instead of manually installing all required components and libraries, you can follow [the quick way to build C/C++ endpoint SDK](#quick-way-to-build).
```
Comment (Polinaa) [inline doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Linux/index.md]: Is link should be `#quick-way-to-build-sdk`?

**Comment by Polinaa** [comment]
Is link should be #quick-way-to-build-sdk in [21 line](https://github.com/kaaproject/kaa/pull/830/files#diff-a6a75140267216dfeb7a54b130ecd835R21)?

**Comment by RostakaGmfun** [comment]
@rasendubi

**Comment by Polinaa** [comment]
LGTM

**Pull request merged**